src/main/java/uk/gov/hmcts/reform/judicialapi/util/RefDataConstants.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.judicialapi.util;
+
+public class RefDataConstants {
+
+    private RefDataConstants() {
```

Comment:
you can add lombok annotation @NoArgsConstructor(Scope= private). this will avoid sonar to complain code coverage of construtor